### PR TITLE
[HDInsight] - Dedicated review - Parameters, commands refinement

### PR DESCRIPTION
**Pull request by idear1203**
## Changes

* BREAKING CHANGE: create: Renamed --storage-default-container to --storage-container and --storage-default-filesystem to --storage-filesystem
* BREAKING CHANGE: application create: 
   1. Changed the --name/-n argument to represent the application name instead of the cluster name and added a separate --cluster-name argument
   2. Renamed --application-type to --type/-t
   3. Renamed --marketplace-identifier to --marketplace-id
   4. Renamed --https-endpoint-access-mode to --access-mode and --https-endpoint-destination-port to --destination-port
   5. Removed --https-endpoint-location, --https-endpoint-public-port, --ssh-endpoint-destination-port, --ssh-endpoint-location and --ssh-endpoint-public-port
* BREAKING CHANGE: resize: Renamed --target-instance-count to --workernode-count/-c
* BREAKING CHANGE: script-action:
   1. Changed --name/-n to represent the name of the script action and added the --cluster-name argument to represent the cluster name
   2. Changed --script-execution-id to --execution-id
   3. Renamed the "show" command to "show-execution-details"
* BREAKING CHANGE: script-action execute: Made parameters for the --roles argument space separated instead of comma separated
* BREAKING CHANGE: script-action list: Removed the --persisted parameter
* create: Enabled the --cluster-configurations argument to accept a path to a local JSON file or a JSON string as the parameter
* script-action list-execution-history: Added this command to list the execution history for all script action executions
* monitor enable:
   1. Enabled the --workspace argument to accept a Log Analytics workspace ID or workspace name as the parameter
   2. Added the --primary-key argument, which is needed if a workspace ID is provided as the parameter 
* Added more examples and updated descriptions for help messages

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

**Comment by yugangw-msft** [comment]
@idear1203 & @tylerfox, let us know if this is ready for review

**Comment by idear1203** [comment]
@yugangw-msft Sorry for the late reply. I was re-generating session records. Now this PR is ready for review.

File: src/azure-cli/azure/cli/command_modules/hdinsight/_help.py [hunk 1]
```diff
@@ -96,22 +117,22 @@
...
 helps['hdinsight monitor'] = """
     type: group
-    short-summary: Manage Azure Monitor logs integration on the HDInsight cluster.
+    short-summary: Manage Azure Monitor logs integration on an HDInsight cluster.
 """
 
 helps['hdinsight monitor enable'] = """
     type: command
-    short-summary: Enables the Azure Monitor logs integration on the HDInsight cluster.
+    short-summary: Enables the Azure Monitor logs integration on an HDInsight cluster.
```
Comment (Juliehzl) [inline src/azure-cli/azure/cli/command_modules/hdinsight/_help.py]: Please use active voice in for all description. `e.g. ``Enables```->```Enable```
The same as others.

File: src/azure-cli/azure/cli/command_modules/hdinsight/_help.py [hunk 2]
```diff
@@ -96,22 +117,22 @@
...
 """
 
 helps['hdinsight monitor disable'] = """
     type: command
-    short-summary: Disables the Azure Monitor logs integration on the HDInsight cluster.
+    short-summary: Disables the Azure Monitor logs integration on an HDInsight cluster.
```
Comment (Juliehzl) [inline src/azure-cli/azure/cli/command_modules/hdinsight/_help.py]: Disables->Disable

File: src/azure-cli/azure/cli/command_modules/hdinsight/_help.py [hunk 3]
```diff
@@ -96,22 +117,22 @@
...
 """
 
 helps['hdinsight monitor show'] = """
     type: command
-    short-summary: Gets the status of Azure Monitor logs integration on the HDInsight cluster.
+    short-summary: Gets the status of Azure Monitor logs integration on an HDInsight cluster.
```
Comment (Juliehzl) [inline src/azure-cli/azure/cli/command_modules/hdinsight/_help.py]: Gets->Get
Please apply for all description message.

File: src/azure-cli/azure/cli/command_modules/hdinsight/_help.py (hunk 1 above)
Comment (idear1203) [reply src/azure-cli/azure/cli/command_modules/hdinsight/_help.py]: Thank you for your suggestions. I will change these verbs.

However please note we cannot change descriptions from SDK. I don't think it is wise to add command description for every command.

File: src/azure-cli/HISTORY.rst [hunk 4]
```diff
@@ -13,6 +13,19 @@ Release History
 * Fixed issue where `az functionapp create` does not accept Standard_RAGRS storage account type.
 * Fixed an issue where az webapp up would fail if run using older versions of python
 
+**HDInsight**
+
+* BREAKING CHANGE: create: Renamed --storage-default-container to --storage-container and --storage-default-filesystem to --storage-filesystem
```
Comment (yugangw-msft) [inline src/azure-cli/HISTORY.rst]: @Juliehzl, please format it to be more readable when you conduct the releasing next week.

**Pull request merged**